tapgarden/planter.go

```diff
@@ -745,15 +746,75 @@ func buildGroupReqs(genesisPoint wire.OutPoint, assetOutputIndex uint32,
 		// already be specified. Use that to derive the key group
 		// signature along with the tweaked key group.
 		if seedling.EnableEmission {
-			if seedling.GroupInternalKey == nil {
+			if seedling.GroupInternalKey == nil &&
```

Comment:
We can cut down on the indentation and line length a bit here by moving this into a new function. This new function can also be more easily tested to isolated the new functionality.